locales/de.json

```diff
@@ -1,574 +1,911 @@
+      },
+      "incomplete": {
+        "singular": "Entsprechende ARIA Kindrolle muss hinzugefügt werden: ${data.values}",
+        "plural": "Entsprechende ARIA Kindrollen müssen hinzugefügt werden: ${data.values}"
+      }
+    },
+    "aria-required-parent": {
+      "pass": "",
+      "fail": {
+        "singular": "Benötigte ARIA Elternrolle nicht vorhanden: ${data.values}",
+        "plural": "Benötigte ARIA Elternrollen nicht vorhanden: ${data.values}"
+      }
+    },
+    "aria-roledescription": {
+      "pass": "aria-roledescription mit einer unterstützten semantischen Rolle verwendet.",
+      "incomplete": "Es sollte überprüft werden ob aria-roledescription von einem Screenreader vorgelesen wird.",
+      "fail": "Das Element muss mit einer Rolle, welche aria-roledescription unterstützt, versehen werden."
+    },
+    "aria-unsupported-attr": {
+      "pass": "ARIA Attribut wird unterstützt",
+      "fail": "ARIA Attribut ist nicht wirklich in Screenreadern und anderen assistiven Technologien unterstützt: ${data.values}"
+    },
+    "aria-valid-attr-value": {
+      "pass": "",
+      "fail": {
+        "singular": "Ungültiger Wert des ARIA Attribtutes ${data.values}",
+        "plural": "Ungültige Werte der ARIA Attribute: ${data.values}"
+      },
+      "incomplete": {
+        "noId": "Verwendete ID im ARIA Attribut existiert nicht auf der Seite: ${data.needsReview}",
+        "ariaCurrent": "Folgendes ARIA Attributwert ist ungültig und wird wie \"aria-current=true\" gesehen: ${data.needsReview}"
+      }
+    },
+    "aria-valid-attr": {
+      "pass": "",
+      "fail": {
+        "singular": "Ungültige ARIA Attribut Name: ${data.values}",
+        "plural": "Ungültige ARIA Attribut Namen: ${data.values}"
+      }
+    },
+    "fallbackrole": {
+      "pass": "Nur ein Wert für role genutzt.",
+      "fail": "Es sollte nur ein Wert für role benutzt werden, da Fallback-Werte in älteren Browsern nicht unterstützt werden."
+    },
+    "has-global-aria-attribute": {
+      "pass": {
+        "singular": "Element hat globales ARIA Attribut: ${data.values}",
+        "plural": "Element hat globale ARIA Attribute: ${data.values}"
+      },
+      "fail": "Das Element hat keine globalen ARIA Attribute."
+    },
+    "has-widget-role": {
+      "pass": "",
+      "fail": "Das Element besitzt keine widget-Rolle."
+    },
+    "invalidrole": {
+      "pass": "",
+      "fail": {
+        "singular": "Folgende Rolle muss eine von den validen ARIA Rollen sein: ${data.values}",
+        "plural": "Folgende Rollen müssen jeweils eine von den validen ARIA Rollen sein: ${data.values}"
+      }
+    },
+    "is-element-focusable": {
+      "pass": "Element ist Fokussierbar.",
+      "fail": "Element ist nicht Fokussierbar."
+    },
+    "no-implicit-explicit-label": {
+      "pass": "Kein Unterschied zwischen dem <label> und dem zugänglichen Namen.",
+      "incomplete": "Überprüfe, dass das <label> kein Teil des ARIA ${data} Feldname ist."
```

Comment:
would `Überprüfe, dass das <label> nicht Teil des ARIA ${data} Feldname ist.` be better?